styles/utils.go

```diff
@@ -118,3 +118,20 @@ func ViewportMin(value int) string {
 func ViewportMax(value int) string {
 	return strconv.Itoa(value) + "vmax"
 }
+
+// Seconds returns a string representation of the given float64 or int
+func Seconds(value interface{}) string {
```

Comment:
I think you should just be able to take a `float64` as the value and return your float64 case. Then I believe `Seconds(10)` or Seconds(10.0)` would still return as `10.00s`.

Reply:
I tried with float64 there, It wasn't accepting 10 as a parameter in that case and function overloading is apparently not possible in go

Reply:
I can confirm it works for `Rem`. Just tried adding these test cases: https://github.com/chasefleming/elem-go/pull/138/files#diff-97855a388ba6def6d1f94ab2c728bead74eb8af8ac30d1a2d6e09c5829336ac8R87

Reply:
I see, What i tried was first creating a variable and then passing which was stemming the confusion
![image](https://github.com/chasefleming/elem-go/assets/76687690/75d645b4-ee09-4577-a9aa-be51b32fb44d)

Directly passing the integer comes across as float. Thanks

Reply:
Yeah, when you define a variable with the shorthand it is inferring the type as int. You'd need to declare it with a float64 type to pass it in from a variable.

Reply:
ha.. fixed in the latest commit. Is it fine to be merged ?